### PR TITLE
nowplaying: decode command output

### PR DESCRIPTION
**Pull request by mayli**

### CODE REVIEW

**Comment by slook** [comment]
Fixes #3039

File: pynicotine/nowplaying.py [hunk 1]
```diff
@@ -300,7 +300,7 @@ def other(self, command):
             return None
 
         try:
-            output = execute_command(command, returnoutput=True, hidden=True)
+            output = execute_command(command, returnoutput=True, hidden=True).decode()
             self.title["nowplaying"] = output
```
Comment (slook) [inline pynicotine/nowplaying.py:304]: Can we have

`            self.title["nowplaying"] = output.decode()`

That is the same effect but easier to read.

**Review by slook — CHANGES_REQUESTED** [review]

**Review by mayli — COMMENTED** [review]
address comment

File: pynicotine/nowplaying.py (hunk 1 above)
Comment (mayli) [reply pynicotine/nowplaying.py]: Done.

**Review by mathiascode — APPROVED** [review]

**Pull request merged**